.github/CODEOWNERS

```diff
@@ -1,12 +1,13 @@
 # Default owners for everything in the repo, unless a later match takes precedence.
-* @mbzomowski @RissyRan @allenwang28
+* @mbzomowski @RissyRan @allenwang28 @ManfeiBai
 
```

Comment:
hi, @RissyRan , do we want to remove this change?

Reply:
You may want to discuss with Yixin and team to see if you are the owner of the XL ML infra. I think @richardsliu currently is the new owner.

Reply:
thanks, be the owner for `dags/mantaray` should cover our request for now, let's get back to this discussion later if needed